frame/staking-rewards/src/lib.rs

```diff
@@ -295,4 +455,10 @@ pub mod pallet {
 			Err("Not implemented".into())
 		}
 	}
+
+	impl<T: Config> Pallet<T> {
+		pub(crate) fn pool_account_id(pool_id: &T::RewardPoolId) -> T::AccountId {
+			T::PalletId::get().into_sub_account(("po", pool_id))
```

Comment:
What is `po`?

Reply:
after recent polkadot upgrade into_sub_account is renamed to into_sub_account_truncating